apps/src/lib/ui/SchoolInfoConfirmationDialog.jsx

```diff
@@ -15,80 +15,102 @@ export const styles = {
 class SchoolInfoConfirmationDialog extends Component {
   static propTypes = {
     schoolName: PropTypes.string,
-    onUpdate: PropTypes.func,
-    onConfirm: PropTypes.func,
-    isOpen: PropTypes.bool.isRequired
+    scriptData: PropTypes.shape({
+      formUrl: PropTypes.string.isRequired,
+      authTokenName: PropTypes.string.isRequired,
+      authTokenValue: PropTypes.string.isRequired,
+      existingSchoolInfo: PropTypes.shape({
```

Comment:
Do we need all of these fields? I see that we need school_id and school_name, but it's not clear we need the other fields in this form.